### ISSUE TITLE
Ledger hw wallet disallows other staking derivation paths

### ISSUE THREAD

**Issue by fallen-icarus** [vacuumlabs/cardano-hw-cli#174]
It appears the only staking key derivation path supported by cardano-hw-cli with Ledger is `1852H/1815H/0H/2/0`.  Using any other index than `0` in the last spot will result in:
```bash
Error: The requested operation failed. Check that your Ledger device is connected, unlocked and with Cardano app running.
Details: DeviceStatusError: Action rejected by Ledger's security policy
```

Is this a limitation in cardano-hw-cli or Ledger? I have not tried this with trezor. The [cip](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0011) just *recommends* only using the `0` index. However, there are legitimate use cases for using more than just the first index, such as asset segregation (there are DApps that delegate spending custody to the staking credential) and [multi-delegaiton](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0018). It seems, right now, the only way around this is to use a different seed phrase for each stake key. Am I missing something that would allow me to use the other indices with a Ledger device?

**Comment by fallen-icarus** [comment]
It seems wallets like eternl just increment the account number (the `0H`) instead of the address index. This works fine for me, too. I'll close this issue.

**Comment by janmazak** [comment]
Multidelegation was implemented and will be included in Ledger app v6.1.2 (soon to be released).

**Comment by fallen-icarus** [comment]
Actually, the account number does not work for my needs. Ledger does not seem to allow signing a transaction with keys from two different account indices. I want to be able to manage multiple stake keys in a single transaction (eg, register one stake key in the same transaction where another has its delegation updated).

@janmazak Will signing with keys from different accounts be possible with the new app version? Or will it still be limited to only keys from the same account? As long as I can create more than one stake key under a single account and I can use several of those keys in the same transaction, I think I would be fine with the latter.
